src/generatorprofile.cpp

```diff
@@ -483,7 +497,7 @@ void GeneratorProfile::GeneratorProfileImpl::loadProfile(GeneratorProfile::Profi
                                       "#include <stdlib.h>\n";
 
         mInterfaceVersionString = "extern const char VERSION[];\n";
-        mImplementationVersionString = "const char VERSION[] = \"0.3.1\";\n";
+        mImplementationVersionString = "const char VERSION[] = \"0.3.2\";\n";
```

Comment:
These changes are a minor version change, not a patch version change.

Reply:
b671d3c2